Readme.md

```diff
@@ -51,15 +51,45 @@ bundled with Dolphin and used if they're not installed on your system. CMake
 will inform you if a bundled library is used or if you need to install any
 missing packages yourself.
 
-### Build Steps:
+### OS X Build Steps:
+1. `mkdir build`
+2. `cd build`
+3. `cmake ..`
+4. `make`
+
+An application bundle will be created in `./Binaries`.
+
+### Linux Global Build Steps:
+
+To install to your system.
+
+1. `mkdir build`
+2. `cd build`
+3. `cmake ..`
+4. `make`
+5. `sudo make install`
+
+### Linux Local Build Steps:
+
+Useful for development as root access is not required.
+
 1. `mkdir Build`
 2. `cd Build`
-3. `cmake ..`
+3. `cmake .. -DLINUX_LOCAL_DEV=true`
 4. `make`
+5. `ln -s ../../Data/Sys Binaries/`
 
-On OS X, an application bundle will be created in `./Binaries`.
+### Linux Portable Build Steps:
 
-On Linux, it's strongly recommended to perform a global installation via `sudo make install`.
+Can be stored on usb and used on different Linux systems.
+Or useful for having multiple distinct dolphin setups for testing/development/TAS.
```

Comment:
USB should be replaced by something like "flash drives", because USB is not used for the storage itself. If you don't make this change, `usb` needs to be replaced with uppercase `USB`.

Dolphin needs to have the first letter capitalized.